src/lib/utils/isFacetRefined.ts

```diff
@@ -0,0 +1,13 @@
+import { AlgoliaSearchHelper } from 'algoliasearch-helper';
+
+export default function isFacetRefined(
+  helper: AlgoliaSearchHelper,
+  facet: string,
+  value: string
+) {
+  if (helper.state.isConjunctiveFacet(facet)) {
```

Comment:
I think this use case fits us good enough, but it means that if there are two refinement lists on the same attribute, one disjunctive, one conjunctive, the isFacetRefined will be wrong.

However, this should be checked, since maybe it's not possible at all to mix

Reply:
![image](https://user-images.githubusercontent.com/499898/88784472-5fbdee00-d190-11ea-9875-e190743dc1fa.png)

okay it's not possible at all at least at the moment.